<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TKNIslemciPencere</class>
 <widget class="QDialog" name="TKNIslemciPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Markası</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Modeli</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Soket Tipi</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>İşlemci Hızı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_2</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maks. Turbo Hızı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_3</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>İşlemci Çekirdek Sayısı</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_4</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>İşlemci Ön Bellek</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_7</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Bellek Türleri</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_9</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Güç Tüketimi</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_10</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Entegre Grafik Kartı</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Var</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>Yok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Soğutucu</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>Var</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>Yok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>İptal Et</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TKNIslemciPencere</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TKNIslemciPencere</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TKNIslemciPencere</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TKNIslemciPencere</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
